modules/op-release-notes-1-17.adoc

```diff
@@ -250,7 +250,7 @@ c
 
 * Before this update, when using {pac}, when you deleted a pipeline run before it was completed, temporary Git authentication secrets remained in the {OCP} namespace. Because of this issue, the quota for secrets could be reached, and in this case new pipeline runs failed to start. With this update, {pac} deletes temporary secrets properly.
 
-* Before this update, when using {pac} with BitBucket, tag-related events were not matched when a pipeline run was configured for `on-target-branch`, for example: `on-target-branch: [refs/tags/*]`. This happened because the BitBucket events payload related to a tag does not contain a `refs/tags` prefix. With this update, {pac} matches the tag events correctly.
+* Before this update, when using {pac} with Bitbucket Cloud, tag-related events were not matched when a pipeline run was configured for `on-target-branch`, for example: `on-target-branch: [refs/tags/*]`. This happened because the BitBucket events payload related to a tag does not contain a `refs/tags` prefix. With this update, {pac} matches the tag events correctly.
```

Comment:
```suggestion
* Before this update, when using {pac} with Bitbucket Cloud, tag-related events were not matched when a pipeline run was configured for `on-target-branch`, for example: `on-target-branch: [refs/tags/*]`. This issue happened because the BitBucket events payload related to a tag does not contain a `refs/tags` prefix. With this update, {pac} matches the tag events correctly.
```
- Avoid ambiguous pronoun references: https://www.ibm.com/docs/en/ibm-style?topic=grammar-pronouns#ambiguous-pronoun-references